zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -1009,6 +1009,48 @@ paths:
             The decision requirements search query failed.
             More details are provided in the response body.
 
+  /decisions/evaluation:
+    post:
+      tags:
+        - Decision
+      summary: Evaluate decision
+      description: |
+        Evaluates a decision.
+        You specify the decision to evaluate either by using its unique KEY (as returned by
```

🔧 I think "key" is still lower-case.
```suggestion
        You specify the decision to evaluate either by using its unique key (as returned by
```